src/Avalonia.Base/Input/TouchDevice.cs

```diff
@@ -59,16 +60,18 @@ public void ProcessRawEvent(RawInputEventArgs ev)
                 else
                 {
                     var settings = AvaloniaLocator.Current.GetRequiredService<IPlatformSettings>();
+                    var doubleClickTime = settings?.GetDoubleTapTime(PointerType.Touch).TotalMilliseconds ?? 500;
+                    var doubleClickSize = settings?.GetDoubleTapSize(PointerType.Touch) ?? new Size(4, 4);
```

Comment:
I think it might be better to have `DefaultPlatformSettings.Instance` so we won't have hardcoded default value in two different places.

Reply:
AS @maxkatz6 mentioned below, it's calling `GetRequiredService` anyway so `settings` shouldn't be null. I copy/pasted this code from `MouseDevice` which uses `GetService`. We should just use `GetRequiredService` I think.